Dockerfile-csi-controller

```diff
@@ -13,15 +13,21 @@
 # limitations under the License.
 
 FROM registry.access.redhat.com/ubi8/python-38:1-77 as builder
-COPY controller/requirements.txt /tmp/
 USER root
 RUN if [[ "$(uname -m)" != "x86"* ]]; then yum install -y rust-toolset; fi
 USER default
 RUN pip3 install --ignore-installed --default-timeout=100 --upgrade pip==21.2.4
 # avoid default boringssl lib, since it does not support z systems
 ENV GRPC_PYTHON_BUILD_SYSTEM_OPENSSL=True
+COPY controller/requirements.txt /tmp/
 RUN pip3 install -r /tmp/requirements.txt
 
+USER root
+COPY controller/scripts/csi_general .
+RUN chmod 755 ./csi_pb2.sh
+RUN ./csi_pb2.sh
+RUN pip3 install .
+USER default
```

Comment:
I don't think this is necessary

Reply:
so why requirements installed again?

Reply:
not sure what you mean. there is no need to install the requirements again.
I just don't see why we need the default user here

Reply:
the conversation is only on the  `USER default`?

Reply:
yes...

Reply:
done